src/rust/otaku_sanctuary/build.sh

```diff
@@ -0,0 +1,37 @@
+# template source build script
+# usage: ./build.sh [source_name/-a]
+
+if [ "$1" != "-a" ]; then
```

Comment:
I'd like it so that `./build.sh` just builds all the sources too, rather than having to specify `-a`.
```suggestion
if [ "$1" != "-a" ] && [ "$1" != "" ]; then
```